src/SurfaceFluxes.jl

```diff
@@ -348,28 +393,181 @@ end
 
 ### Generic terms
 
+function b_star_from_ρ(param_set, 
+                       p_sfc, ρ_sfc, 
+                       T_sfc, T_in,
+                       ::FVScheme
+                       )
+    ρ_star = compute_physical_scale_coeff(uf, Δz, z_0, L_MO)*(ρ_in - ρ_sfc)
+    b_star = -grav * ρ_star/ ρ_in
+    return b_star
+end
+
+function b_star_from_T(param_set, 
+                       p_sfc, ρ_sfc, 
+                       T_sfc, T_in,
+                       ::DGScheme
+                       )
+    ρ_star = compute_physical_scale_coeff(uf, Δz, z_0, L_MO)*(ρ_in - ρ_sfc)
+    b_star = -grav * ρ_star/ ρ_in
+    return b_star
+end
+
+"""
+    b_star_from_θ(param_set, θ_scale, qt_scale, qt_star)
+Returns b⋆ (calculated from potential temperature scale)
+If θ is θᵥ then moisture is accounted for. 
+"""
+
+function b_star_from_θ(param_set, 
+                               u_star, 
+                               θ_star, θ_scale, 
+                               qt_scale, qt_star)
+    FT = typeof(θ_scale)
+    grav::FT = CPP.grav(param_set)
+    von_karman_const::FT = CPSGS.von_karman_const(param_set)
+    ϵ::FT = CPP.molmass_ratio(param_set)
+    return (1 + (ϵ-1)*qt_scale)*θ_star + (ϵ-1)*θ_scale*qt_star
```

Comment:
The naming 'θ_scale' and 'θ_star' is confusing. 'θ_scale' is the value of θ right ?

Reply:
Changed per Nishizawa notation \theta_ave and qt_ave